config/cesm/machines/config_machines.xml

```diff
@@ -352,6 +352,90 @@ This allows using a different mpirun command to launch unit tests
     </resource_limits>
   </machine>
 
+   <machine MACH="casper">
+    <DESC>NCAR GPU platform, os is Linux, 32 pes/node, batch system is slurm</DESC>
+    <NODENAME_REGEX>casper*</NODENAME_REGEX>
+    <!-- MPT sometimes timesout at model start time, the next two lines cause
+    case_run.py to detect the timeout and retry FORCE_SPARE_NODES times -->
+    <MPIRUN_RETRY_REGEX>openmpi: Launcher network accept (MPI_LAUNCH_TIMEOUT) timed out</MPIRUN_RETRY_REGEX>
```

Comment:
Have you tested this?  This code is mpt specific along with several other settings here - can you remove them? (The mpt specific stuff).

Reply:
This change is part of a porting of MOM6 using OpenACC by NCAR. We have checked that this allows for the code to run on our single node on Casper, and we are currently in the process of verifying the results with the PGI compiler on Cheyenne.
We have not run the regression test suite with these changes.